tests/isolated/generator.cpp

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
I don't understand how this is more "isolated" than the other tests?  It's still linking to the library in the same way?